ziosql/src/main/resources/init.sql

```diff
@@ -0,0 +1,73 @@
+DROP TABLE IF EXISTS metro_line;
+DROP TABLE IF EXISTS metro_system;
+DROP TABLE IF EXISTS city;
+
+CREATE TABLE city(
```

Comment:
can't you use the same schema/data as the others? (there is a dependency on `common` so I suppose this should work ...)

Reply:
done